### PR TITLE
Starplat 1715

### PR DESCRIPTION
**Pull request by SuitedJoe**
Just updating to add sync to developer docs since we're going to continue to support this.

### CODE REVIEW

File: .github/workflows/update_readme.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout this repo
+        uses: actions/checkout@v3
+
+      #Token used is a token associated with Joe Malecki; if he's no longer with the company than this may fail
+      - name: Copy updated Readme to Documentation Repo
+        uses: andstor/copycat-action@v3
+        with:
+          personal_token: ${{ secrets.JOE_GITHUB_ACCESS_TOKEN }}
```
Comment (vanceingalls) [inline .github/workflows/update_readme.yml:19]: `secrets.PERSONAL_TOKEN` doesn't work here?

Comment (vanceingalls) [inline .github/workflows/update_readme.yml:19]: We should probably move an existing `PERSONAL_TOKEN` to org level so they can be shared across all repos. Are you sure want to use your token here?

Comment (SuitedJoe) [reply .github/workflows/update_readme.yml:19]: Can we do that? I'm not familiar with what all our Github token options are and if they'd work. I was thinking I'd originally make a tech debt ticket and try and fix it at a later sprint. Just trying to get all the documentation together on our dev docs portal.

Comment (SuitedJoe) [reply .github/workflows/update_readme.yml:19]: I'll need to fix the token for actually 3 other repos, unity login, unreal login, and Documentation as they all use my user token. So fixing it here alone, would still be an incomplete fix.

Comment (SuitedJoe) [reply .github/workflows/update_readme.yml:19]: Created Ticket with Label "TechDebt" https://stardustgg.atlassian.net/browse/STARPLAT-2200

**Review by vanceingalls — APPROVED** [review]
LGTM

**Pull request merged**